components/brave_rewards/resources/extension/brave_rewards/components/app.tsx

```diff
@@ -98,21 +99,29 @@ export class RewardsPanel extends React.Component<Props, State> {
   }
 
   render () {
-    const { walletCreateFailed, walletCreated } = this.props.rewardsPanelData
+    const { enabledMain, walletCreateFailed, walletCreated } = this.props.rewardsPanelData
+
+    if (!walletCreated) {
+      return (
+        <PanelWelcome
+          error={walletCreateFailed}
+          creating={this.state.creating}
+          variant={'two'}
+          optInAction={this.onCreate}
+          optInErrorAction={this.onCreate}
+          moreLink={this.openRewards}
+        />
+      )
+    }
 
     return (
       <>
         {
-          !walletCreated
-          ? <PanelWelcome
-            error={walletCreateFailed}
-            creating={this.state.creating}
-            variant={'two'}
-            optInAction={this.onCreate}
-            optInErrorAction={this.onCreate}
-            moreLink={this.openRewards}
-          />
-          : <Panel windowId={this.state.windowId} />
+          enabledMain
+          ? <Panel windowId={this.state.windowId} />
+          : <div style={{ width: '330px' }}>
```

Comment:
I think the width should have originally been defined in the component how I originally had it, without defining container the panel appears super slim.